app/client/components/callCentreNumbers.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Accordion } from "./accordion";
+
+export const CallCentreNumbers = () => (
+  <Accordion>
+    <div title="United Kingdom, Europe and rest of world">
+      <div>
+        <b>+ 44 (0) 330 333 6790</b>
```

Comment:
We might want to match the true font-weight instead of just using a `<b>` tag here, as the fonts will have one specifically for bold.